internal/config/config.go

```diff
@@ -111,18 +114,43 @@ func InitLog() {
 	if err := os.MkdirAll(filepath.Join(GetHomeDir(), "log"), 0755); err != nil {
 		panic(err)
 	}
-	file, err := os.OpenFile(filepath.Join(GetHomeDir(), "log", "opengist.log"), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		panic(err)
-	}
 
 	var level zerolog.Level
-	level, err = zerolog.ParseLevel(C.LogLevel)
+	level, err := zerolog.ParseLevel(C.LogLevel)
 	if err != nil {
 		level = zerolog.InfoLevel
 	}
 
-	multi := zerolog.MultiLevelWriter(zerolog.NewConsoleWriter(), file)
+	var logWriters []io.Writer
+	logOutputTypes := utils.RemoveDuplicates[string](
+		strings.Split(strings.ToLower(C.LogOutput), ","),
+	)
+	for _, logOutputType := range logOutputTypes {
+		logOutputType = strings.TrimSpace(logOutputType)
+		if !utils.SliceContains([]string{"stdout", "file"}, logOutputType) {
+			defer func() { log.Warn().Msg("Invalid log output type: " + logOutputType) }()
```

Comment:
Used defer for these logs so that they get logged after the logger is initialized